src/reanimated2/layoutReanimation/web/animationParser.ts

```diff
@@ -0,0 +1,62 @@
+export interface ReanimatedWebTransformProperties {
+  translateX?: string;
+  translateY?: string;
+  rotate?: string;
+  rotateX?: string;
+  rotateY?: string;
+  scale?: number | string;
+  scaleX?: number;
+  scaleY?: number;
+  perspective?: string;
+  skew?: string;
+  skewX?: string;
+}
+
+interface AnimationStyle {
+  opacity?: number;
+  transform?: ReanimatedWebTransformProperties[];
+}
+
+export interface AnimationData {
+  name: string;
+  style: Record<number, AnimationStyle>;
+  duration: number;
+}
+export interface TransitionData {
+  translateX: number;
+  translateY: number;
+  scaleX: number;
+  scaleY: number;
+  reversed?: boolean;
+}
+
+export function parseAnimationObjectToKeyframe(animationObject: AnimationData) {
```

Comment:
```suggestion
export function convertAnimationObjectToKeyframes(animationObject: AnimationData) {
```

Reply:
Done in [1b57aa1](https://github.com/software-mansion/react-native-reanimated/pull/4805/commits/1b57aa1a29d6a706a96ee84f23fc8a426d27ec22).